ArchiSteamFarm/Json/Steam.cs

```diff
@@ -390,9 +390,26 @@ internal class NumberResponse {
 
 		[SuppressMessage("ReSharper", "ClassCannotBeInstantiated")]
 		internal sealed class RedeemWalletResponse : EResultResponse {
+			internal sealed class Wallet {
+				[JsonProperty(PropertyName = "currency", Required = Required.Always)]
+				internal readonly ulong Amount;
+
+				[JsonProperty(PropertyName = "currencycode", Required = Required.Always)]
+				internal readonly ECurrencyCode CurrencyCode;
+
+				// Deserialized from JSON
+				private Wallet() { }
+			}
+
 			[JsonProperty(PropertyName = "detail", Required = Required.DisallowNull)]
 			internal readonly EPurchaseResultDetail? PurchaseResultDetail;
 
+			[JsonProperty(PropertyName = "currencycode", Required = Required.DisallowNull)]
+			internal readonly Wallet RedeemingCodeWallet;
```

Comment:
I like to call classes `InternalWallet` in this case and use `Wallet` as the name of property. If there is already such thing in `Steam.cs`, stick to it.